scipy/integrate/_quadrature.py

```diff
@@ -399,8 +399,8 @@ def _basic_simpson(y, start, stop, x, dx, axis):
     slice2 = tupleset(slice_all, axis, slice(start+2, stop+2, step))
 
     if x is None:  # Even-spaced Simpson's rule.
-        result = np.sum(dx/3.0 * (y[slice0]+4*y[slice1]+y[slice2]),
-                        axis=axis)
+        result = np.sum(y[slice0]+4*y[slice1]+y[slice2], axis=axis)
```

Comment:
Since you did add some spaces line 415 to 417, I would suggest doing the following

```suggestionc
        result = np.sum(y[slice0] + 4 * y[slice1] + y[slice2], axis=axis)
```

Reply:
Changed to `result = np.sum(y[slice0] + 4*y[slice1] + y[slice2], axis=axis)`